File: VB/DXDocumentUIServiceSample/Common/StringAggregationConverterExtension.vb
Imports System
Imports System.Windows.Markup

Namespace DXDocumentUIServiceSample.Common
	Public Class StringAggregationConverterExtension
		Inherits MarkupExtension

		Public Overrides Function ProvideValue(ByVal serviceProvider As IServiceProvider) As Object
			Return New StringAggregationConverter()
		End Function
	End Class
End Namespace

File: CS/DXDocumentUIServiceSample/Common/StringAggregationConverterExtension.cs
using System;
using System.Windows.Markup;

namespace DXDocumentUIServiceSample.Common {
    public class StringAggregationConverterExtension : MarkupExtension {
        public override object ProvideValue(IServiceProvider serviceProvider) {
            return new StringAggregationConverter();
        }
    }
}
